setup/sql/dbupdate_custom_renameme.php

```diff
@@ -0,0 +1,7 @@
+<#1>
+<?php
+	if($ilDB->tableExists('svy_qst_oblig'))
+		$ilDB->dropTable('svy_qst_oblig');
+	if($ilDB->tableExists('svy_qst_oblig_seq'))
+		$ilDB->dropTable('svy_qst_oblig_seq');
+?>
```

Comment:
@alex40724 I forced the push for the "dbupdate_custom.php" file with the migration. I also deleted the old "dbupdate_custom_renameme.php" file.